tslint.json

```diff
@@ -55,7 +55,7 @@
       "ignore-params"
     ],
     "no-misused-new": true,
-    "no-non-null-assertion": true,
+    "no-non-null-assertion": false,
```

Comment:
There might even be new features to tslint that we don't have setup since they have done several releases recently.